src/knock.ts

```diff
@@ -13,6 +14,7 @@ class Knock {
 
   readonly feeds = new FeedClient(this);
   readonly preferences = new Preferences(this);
+  readonly users = new UserClient(this);
```

Comment:
When I was writing the linear ticket one thought I had was should this be _user_ (singular) given it only operates on the current user? wdyt?

Reply:
Okay yeah that makes sense, I'll update that

Reply:
Or @cjbell, you don't think it should just be `knockClient.currentUser. ...` do you? just to be very clear?

Reply:
I think I prefer `user` tbh

Reply:
okay don